### PR TITLE
Sigmoid

### PR DESCRIPTION
**Pull request by robert-wagner**
This does not work because when multiplying tensors by themselves we get a ('All arguments should share the same child type.', [<class 'syft.core.frameworks.torch.tensor.LongTensor'>, <class 'syft.core.frameworks.torch.tensor._MPCTensor'>]) error. I have no clue how to debug this in the current pysyft. @iamtrask @LaRiffle If you could take a look at it that would be great. This is the last step

### CODE REVIEW

File: examples/other/experimental/new spdz demo.ipynb [hunk 1]
```diff
@@ -43,10 +43,29 @@
   {
    "cell_type": "code",
    "execution_count": 3,
+<<<<<<< HEAD:examples/other/experimental/new spdz demo.ipynb
```
Comment (jvmancuso) [inline examples/other/experimental/new spdz demo.ipynb]: looks like a merge conflict in your ipynb 😬

**Comment by iamtrask** [comment]
Should I keep this open?

**Comment by robert-wagner** [comment]
Changes have been implented elsewhere
